proto/enums.proto

```diff
@@ -261,6 +261,9 @@ enum GpioAction
 
     BUTTON_PRESS_INPUT_REVERSE   = 69;
     SUSTAIN_FOCUS_MODE           = 70;
+
+    SUSTAIN_4_WAY_MODE = 71;
```

Comment:
And this one we can simplify down to SUSTAIN_4_8_WAY_MODE = 71; since we can use a single GpioAction::